Kentico.AcceleratedMobilePages/AmpFilter.cs

```diff
@@ -69,8 +69,8 @@ private string AppendAmpHtmlLink(string finalHtml)
         {
             string ampLink = (CMSHttpContext.Current.Request.IsSecureConnection ? Constants.P_HTTPS : Constants.P_HTTP) +
                              Settings.AmpFilterDomainAlias +
-                             (DocumentContext.CurrentPageInfo.DocumentUrlPath ?? DocumentContext.CurrentAliasPath) +
-                             Settings.CmsFriendlyUrlExtension;
+                             (!String.IsNullOrEmpty(DocumentContext.CurrentPageInfo.DocumentUrlPath) ? DocumentContext.CurrentPageInfo.DocumentUrlPath : DocumentContext.CurrentAliasPath) +
```

Comment:
This is getting longer and it is used on two places... Could you please refactor the
`(!String.IsNullOrEmpty(DocumentContext.CurrentPageInfo.DocumentUrlPath) ? DocumentContext.CurrentPageInfo.DocumentUrlPath : DocumentContext.CurrentAliasPath)`
to a method?